spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
```

Comment:
- [ ] Good job on implementing all requested tests for the `posts#index` action, Kindly also include all tests for the `posts#show` action.